src/main/scala/mesosphere/marathon/core/appinfo/AppInfoConfig.scala

```diff
@@ -0,0 +1,21 @@
+package mesosphere.marathon
+package core.appinfo
+
+import org.rogach.scallop.ScallopConf
+
+trait AppInfoConfig extends ScallopConf {
+
+  lazy val appInfoModuleExecutionContextSize = opt[Int](
```

Comment:
Let's base the default as `Runtime.getRuntime().availableProcessors() * 2`? (or... just # of availableProcessors?)